Source/OnlineSubsystemEpic/Private/OnlineSubsystemEpic.cpp

```diff
@@ -382,8 +382,8 @@ bool FOnlineSubsystemEpic::Shutdown()
 	DESTRUCT_INTERFACE(IdentityInterface);
 	DESTRUCT_INTERFACE(SessionInterface);
 	DESTRUCT_INTERFACE(UserInterface);
-	DESTRUCT_INTERFACE(this->PresenceInterface);
-
+	DESTRUCT_INTERFACE(PresenceInterface);
+	
```

Comment:
The changes here were unecessary, so please remove the whitespace and re-add the removed `this->`.